e2e/suites/argocd/suite_test.go

```diff
@@ -32,8 +32,8 @@ import (
 var _ = SynchronizedBeforeSuite(func() []byte {
 	cfg := &addon.Config{}
 	cfg.KubeConfig, cfg.KubeClientSet, cfg.CRClient = util.NewConfig()
-	installArgo(cfg)
-	installESO(cfg)
+	installArgo()
```

Comment:
why did you decide to no longer pass the cfg here? is it not needed anymore?

Reply:
yeah, the cfg is no longer needed, i've cleaned that up :broom: 
The `InstallGlobalAddon()` / `framework.Install()` create/maintain a config on their own, there's no need to pass it from the outside.